Submitted By: Jeremy Utley (jeremy@linuxfromscratch.org)
Date: 2004-05-20
Initial Package Version: 0.40.3
Upstream Status: Not Submitted
Origin: Self Created
Description: Fixes syntax issues with GCC 3.4

$LastChangedBy$
$Date$

diff -aur Guppi-0.40.3/libguppitank/guppi-object-barchart.c 
Guppi-0.40.3-new/libguppitank/guppi-object-barchart.c
--- Guppi-0.40.3/libguppitank/guppi-object-barchart.c	2002-01-07 22:28:53.000000000 -0800
+++ Guppi-0.40.3-new/libguppitank/guppi-object-barchart.c	2004-04-13 15:00:04.587307405 -0700
@@ -701,6 +701,7 @@
     break;
 
   default:
+  break;
   }
 }
 
@@ -733,6 +734,7 @@
     break;
 
   default:
+  break;
   }
 }
 
diff -aur Guppi-0.40.3/src/demo.c Guppi-0.40.3-new/src/demo.c
--- Guppi-0.40.3/src/demo.c	2002-01-07 22:28:57.000000000 -0800
+++ Guppi-0.40.3-new/src/demo.c	2004-04-13 14:58:41.413566037 -0700
@@ -451,6 +451,7 @@
     compass_pos = GUPPI_NORTH;
     break;
   default:
+  break;
   }
 
   guppi_element_state_set (state,
diff -aur Guppi-0.40.3/src/libguppiplot/guppi-layout-constraint.c Guppi-0.40.3-new/src/libguppiplot/guppi-layout-constraint.c
--- Guppi-0.40.3/src/libguppiplot/guppi-layout-constraint.c	2002-01-18 18:45:21.000000000 -0800
+++ Guppi-0.40.3-new/src/libguppiplot/guppi-layout-constraint.c	2004-04-13 14:57:20.328315976 -0700
@@ -165,6 +165,7 @@
 
   default:
     /* Fall through, do nothing. */
+  break;
   }
 
   ti = term_info_new (type, factor, geom);
diff -aur Guppi-0.40.3/src/libguppiplot/guppi-root-group-item.c Guppi-0.40.3-new/src/libguppiplot/guppi-root-group-item.c
--- Guppi-0.40.3/src/libguppiplot/guppi-root-group-item.c	2002-01-07 22:31:08.000000000 -0800
+++ Guppi-0.40.3-new/src/libguppiplot/guppi-root-group-item.c	2004-04-13 14:58:00.977415073 -0700
@@ -548,6 +548,7 @@
     return motion_notify_event (root, &(ev->motion));
 
   default:
+  break;
   }
 
   return FALSE;
@@ -777,6 +778,7 @@
   case ROOT_GROUP_RESIZE_NONE:
   default:
     /* insure we do nothing */
+  break;
   }
 
   if (please_resize_canvas)
diff -aur Guppi-0.40.3/src/libguppiplot/guppi-text-block.c Guppi-0.40.3-new/src/libguppiplot/guppi-text-block.c
--- Guppi-0.40.3/src/libguppiplot/guppi-text-block.c	2001-11-26 12:00:17.000000000 -0800
+++ Guppi-0.40.3-new/src/libguppiplot/guppi-text-block.c	2004-04-13 14:56:55.358397900 -0700
@@ -900,6 +900,7 @@
 
     default:
       /* Do nothing. */
+    break;
     }
 
     render_stack_evolve (stack, tt);
diff -aur Guppi-0.40.3/src/libguppiplot/guppi-view-interval.c Guppi-0.40.3-new/src/libguppiplot/guppi-view-interval.c
--- Guppi-0.40.3/src/libguppiplot/guppi-view-interval.c	2002-01-18 18:45:21.000000000 -0800
+++ Guppi-0.40.3-new/src/libguppiplot/guppi-view-interval.c	2004-04-13 14:56:16.518858049 -0700
@@ -227,6 +227,7 @@
 #endif
 
   default:
+  break;
   }
 
   return TRUE;
